client/src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from "./logo.svg";
 import "./App.css";
+import Sponsors from "./components/Sponsors/Sponsors";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code className="underline">src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <></>;
```

Comment:
You should atleast leave a working sample usage in this file. Readme is not clear enough

Reply:
I have updated the PR with a working sample of the component in App.js. Please check it.